src/suites/cts/validation/createBindGroupLayout.spec.ts

```diff
@@ -63,30 +63,6 @@ g.test('negative binding index', async t => {
   });
 });
 
-// TODO: Update once https://github.com/gpuweb/gpuweb/issues/405 is decided.
-g.test('Visibility of bindings cannot be None', async t => {
```

Comment:
It can still be 0 though so I think we should keep the test.

Reply:
How can it be 0 it 0 is not a valid value?
See https://chromium-review.googlesource.com/c/chromium/src/+/1845762/1/third_party/blink/renderer/modules/webgpu/gpu_shader_stage.idl

Reply:
It is just a number, you can still pass 0. The only thing that changed is that we don't have a constant for 0 because it is understood that NONE is always 0 for bitflags.

I need to send the TL;DR for the F2F discussions but IIRC we decided that 0 was a valid thing to have, just not a useful constant to expose.

Reply:
Understood, I've renamed test then. Thanks!